docs/development/upgrading-node.md

```diff
@@ -2,14 +2,15 @@
 
 ## Discussion
 
-One upgrade issue is building all of Electron with a single copy
-of V8 to ensure compatibility. This is important because
-upstream Node and [libchromiumcontent](upgrading-chromium.md)
-both use their own versions of V8.
+Chromium and Node.js both depend on V8, and Electron contains
+only a single copy of V8, so it's important to ensure that version
```

Comment:
> so it's important to ensure that version 

should be 

> so it's important to ensure that the version

Reply:
'That version' refers back to the single copy of v8, so I don't think this is incorrect.

Could be clearer to make that reference more explicit, e.g. "so it's important to ensure the single copy of v8 is compatible..."